packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -70,62 +99,71 @@ for i in ansible ansible_galaxy; do
 done
 
 if [ -f %{foreman_proxy_dir}/.ansible.cfg ] && [ ! -L %{foreman_proxy_dir}/.ansible.cfg ]; then
-  mv %{foreman_proxy_dir}/.ansible.cfg %{_sysconfdir}/foreman-proxy/ansible.cfg
+  mv %{foreman_proxy_dir}/.ansible.cfg %{_root_sysconfdir}/foreman-proxy/ansible.cfg
```

Comment:
I wonder if we need an additional macro for `%{_root_sysconfdir}/foreman-proxy` given its repetition. It could also be used in `foreman_proxy_settingsd_dir`

Reply:
I considered it. I saw it used in two plugins I think which left me on the fence.